de-lib/src/main/java/org/iplantc/de/theme/base/client/admin/ontologies/OntologyDisplayStrings.properties

```diff
@@ -31,8 +31,8 @@ ontologyDeleted = Deleted {0} successfully
 rootColumnLabel = Iri(s)
 hierarchyColumnLabel = Root(s)
 confirmDeleteHierarchy = Are you sure you want to delete the `{0}` hierarchy?<BR><BR>NOTE: You can always add it back by re-saving any root hierarchy to which it is a child.
-confirmDeleteAppTitle = Delete App
-confirmDeleteAppWarning = Are you sure you want to delete the app `{0}`?
+confirmDeprecateAppTitle = Deprecate App
+confirmDeprecateAppWarning = Are you sure you want to deprecate the app `{0}`?
```

Comment:
I see this label updated from `delete` to `deprecate` 👍 but I still don't think we need to rename the internal property name.